### PR TITLE
Dummy Test Run Index Screen

### PR DESCRIPTION
**Pull request by mmcbee1**
Clicking a test run takes the user to a placeholder test run index screen, which currently shows the name and ID of the chosen test run

### CODE REVIEW

File: Mobile Test Runner for Jama/Mobile Test Runner for Jama/TestListViewController.swift [hunk 1]
```diff
@@ -169,6 +170,15 @@ extension TestListViewController: UITableViewDelegate, UITableViewDataSource {
         }
         //TODO implement what happens when the user taps a test run
```
Comment (dwagg) [inline Mobile Test Runner for Jama/Mobile Test Runner for Jama/TestListViewController.swift]: Can we remove this comment?

File: Mobile Test Runner for Jama/Mobile Test Runner for Jama/TestListViewController.swift [hunk 2]
```diff
@@ -17,6 +17,7 @@ class TestListViewController: UIViewController {
     var projectId = -1
     var selectedPlanId = -1
     var selectedTestCycleId = -1
+    var selectedRunId = -1
```
Comment (dwagg) [inline Mobile Test Runner for Jama/Mobile Test Runner for Jama/TestListViewController.swift]: Is this variable being used anywhere?

**Comment by dwagg** [comment]
Can you move the views in the storyboard around so that we are not seeing the segue from the TestListView go under the TestStepView to the TestStepIndexView

File: Mobile Test Runner for Jama/Mobile Test Runner for Jama/TestListViewController.swift (hunk 2 above)
Comment (mmcbee1) [reply Mobile Test Runner for Jama/Mobile Test Runner for Jama/TestListViewController.swift]: Nope! I thought I would need it but didn't. Removed.

File: Mobile Test Runner for Jama/Mobile Test Runner for Jama/TestListViewController.swift (hunk 1 above)
Comment (mmcbee1) [reply Mobile Test Runner for Jama/Mobile Test Runner for Jama/TestListViewController.swift]: Absolutely. Done.

**Comment by mmcbee1** [comment]
Updated according to David's comments.

**Review by dwagg — APPROVED** [review]
Branch looks good, all requested changes were addressed.

**Pull request merged**